fastdds_python/test/api/test_datareader.py

```diff
@@ -961,15 +962,173 @@ def test_read():
             topic, fastdds.DATAREADER_QOS_DEFAULT)
     assert(datareader is not None)
 
-    #data_seq = test_complete.CompleteTestTypeSeq()
-    #info_seq = fastdds.SampleInfoSeq()
-    #assert(fastdds.ReturnCode_t.RETCODE_NO_DATA == datareader.read(
-    #    data_seq, info_seq))
+    data_seq = test_complete.CompleteTestTypeSeq()
+    info_seq = fastdds.SampleInfoSeq()
+    assert(fastdds.ReturnCode_t.RETCODE_NO_DATA == datareader.read(
+        data_seq, info_seq, fastdds.LENGTH_UNLIMITED,
+        fastdds.ANY_SAMPLE_STATE, fastdds.ANY_VIEW_STATE,
+        fastdds.ANY_INSTANCE_STATE))
+
+    publisher = participant.create_publisher(fastdds.PUBLISHER_QOS_DEFAULT)
+    assert(publisher is not None)
+    datawriter = publisher.create_datawriter(
+            topic, fastdds.DATAWRITER_QOS_DEFAULT)
+    assert(datawriter is not None)
+    sample = test_complete.CompleteTestType()
+    sample.int16_field(255)
+    assert(datawriter.write(sample))
+
+    assert(datareader.wait_for_unread_message(fastdds.Duration_t(5, 0)))
+    assert(fastdds.ReturnCode_t.RETCODE_OK == datareader.read(
+        data_seq, info_seq, fastdds.LENGTH_UNLIMITED,
+        fastdds.ANY_SAMPLE_STATE, fastdds.ANY_VIEW_STATE,
+        fastdds.ANY_INSTANCE_STATE))
+    assert(1 == len(data_seq))
```

Comment:
Also check length of `info_seq`. Same elsewhere

Reply:
Done